lib/services/irc.rb

```diff
@@ -137,6 +137,31 @@ def irc_puts(command, debug_command=command)
     writable_irc.puts command
   end
 
+  def irc_realname
+    repo_owner = payload["repository"]["owner"]["name"]
+    repo_name = payload["repository"]["name"]
+
+    "GitHub IRCBot - #{repo_owner}" + (repo_public? ? "/#{repo_name}" : "")
+  end
+
+  def repo_public?
```

Comment:
Do we need this method? Looking at https://developer.github.com/v3/activity/events/types/ it seems like every event type has a repository hash with a `private` key. So, it would look something like `private = payload["repository"]["private"]`. 

@kdaigle, @jasonrudolph - Do you all think the `private` field will exist for all events that contain a `repository`?